include/tester/vtcp/tcp/TCPServer.hpp

```diff
@@ -73,4 +84,8 @@ class TCPServer
     }
 
     TCPListener _listener;
+    std::function<bsoncxx::document::value(int)> _startAutoTest;
```

Comment:
absolutely no reason to use this here in this class

Reply:
done